lib/pf/api.pm

```diff
@@ -23,6 +23,7 @@ use pf::config::cached;
 use pf::ConfigStore::Interface();
 use pf::ConfigStore::Pf();
 use pf::iplog();
+use pf::fingerbank;
```

Comment:
This was done for pf::api to avoid exporting methods into the pf::api namespace.
To avoid people from calling methods that were not meant to be exported.
However it is not needed anymore since we have the Public attribute for all functions that are exported.